src/aap_eda/api/views/auth.py

```diff
@@ -116,9 +118,10 @@ def post(self, request: Request, *args, **kwargs):
 class RoleViewSet(
     viewsets.ReadOnlyModelViewSet,
 ):
-    queryset = models.Role.objects.order_by("id")
+    queryset = RoleDefinition.objects.order_by("id")
     filter_backends = (DjangoFilterBackend,)
     filterset_class = filters.RoleFilter
+    deprecated = True
```

Comment:
Note that, I don't think this does anything. We have some tooling in django-ansible-base that will do something with this, but it's not hooked up to this.